presto-main/src/main/java/com/facebook/presto/operator/aggregation/builder/InMemoryHashAggregationBuilder.java

```diff
@@ -171,9 +170,14 @@ public double getExpectedHashCollisions()
     }
 
     @Override
-    public ListenableFuture<?> isBlocked()
+    public ListenableFuture<?> startMemoryRevoke()
+    {
+        return null;
```

Comment:
shouldn't it rather throw? If we're here it means someone wants some memory back and we're giving none. It's an error.